src/Showoff.jl

```diff
@@ -2,10 +2,10 @@ module Showoff
 
 using Dates
 
-if isdefined(Base, :Grisu)
-    include("grisu.jl")
-else
+if isdefined(Base, :Ryu)
```

Comment:
I believe if Ryu exists, it should instead go to Ryu as now 1.5.3 users can now also enjoy new updates